src/Message/Communicate.cpp

```diff
@@ -87,17 +87,6 @@ void Communicate::initialize(const mpi3::environment& env)
   d_ncontexts = comm.size();
   d_groupid   = 0;
   d_ngroups   = 1;
-#ifdef __linux__
-  for (int proc = 0; proc < OHMMS::Controller->size(); proc++)
-  {
-    if (OHMMS::Controller->rank() == proc)
-    {
-      fprintf(stderr, "Rank = %4d  Free Memory = %5zu MB\n", proc, (freemem() >> 20));
```

Comment:
Please remove `#include "Platforms/Host/sysutil.h"` which is only need by freemem.

Reply:
I will take care of this as I'm preparing another similar change.